### ISSUE TITLE
auto-gen-config doesn't patch %r{/} warning correctly

### ISSUE THREAD

**Issue by tmorris-fiksu** [rubocop/rubocop#847]
To reproduce, create the file `sample.rb` included below.  Run `rubocop --auto-gen-config`.  `rubocop --config rubocop-todo.yml` gives the following warning:

```
...
sample.rb:2:9: C: Use %r only for regular expressions matching more than 2 '/' characters.
y.gsub!(%r{abc/xyz}, "A/Z")
        ^^^^^^^^^^^

1 file inspected, 1 offence detected
$ cat sample.rb
y = "123/abc/xyz/ijk"
y.gsub!(%r{abc/xyz}, "A/Z")
puts y
$ rubocop -V
0.18.1 (using Parser 2.1.5, running on ruby 1.9.3 x86_64-darwin12.5.0)
```

The generated `rubocop-todo.yml` includes:

``` bash
# Offence count: 1
RegexpLiteral:
  MaxSlashes: 2
```

It should be `MaxSlashes: 1` to avoid the warning.
